scylla-check-gossiper-generation/scylla-check-gossiper-generation.sh

```diff
@@ -2,31 +2,62 @@
 #
 # This script checks the node's gossiper generation and produces an alert if it is older 
 # than 1 year.
+# !
+# !scylla-check-gossiper-generation.sh [-h] [-n <nodetool command, by default 'nodetool'>]
+# !
 
-set -e
+usage() {
+    cat $0 | grep "^# !" | cut -d"!" -f2-
```

Comment:
wouldn't it be easier just to echo the single line? ;-)
but this works too, so just nitpicking